app/views/incidents/edit.html.haml

```diff
@@ -9,24 +9,20 @@
   - else
     Completing
   Incident Report
-%hr
 - if @current_user.staff?
+  %hr
   %h2 Incident Details
   = render 'incidents/form', incident: @incident
-- if @current_user == @incident.driver || @current_user.staff?
-  %hr
-  %h2 Driver Report
-  = render 'incident_reports/form',
-    report: @incident.driver_incident_report
 %hr
+- if @current_user == @incident.driver || @current_user.staff?
```

Comment:
Just a question about incidents: can anyone who is a staff edit the driver account of an incident?

Reply:
(If I understand this properly)

Reply:
Correct.